macros/edr/system/system_utils/get_config_var.sql

```diff
@@ -63,7 +63,8 @@
       'drop_failure_percent_threshold': none
     },
     'include_other_warehouse_specific_columns': false,
-    'fail_on_zero': false
+    'fail_on_zero': false,
+    'dismiss_on_run_end': false
```

Comment:
Maybe call this `disable_on_run_end` just to be consistent with other flags

Reply:
Also - maybe call it `disable_elementary_on_run_end` (a bit clearer when referenced from a different project)